reports/California.md

```diff
@@ -412,6 +417,50 @@ id: ca-losangeles-17
 * https://abc7.com/society/homeless-man-hit-in-face-with-rubber-bullet-amid-la-protest-witnesses-say/6234476/
 
 
+### Police fire into backs of protestors | June 2nd
+
+A handful of protestors are seen walking away from police with their hands up. Police shoot at the protestors repeatedly for the duration of the video, about 40 seconds.
+
+tags: shoot, projectile
+
+id: ca-losangeles-27
+
+**Links**
+
+* https://twitter.com/MattMcGorry/status/1268455032032133122
+
+
+### Police arrest passing motorists, allegedly for curfew violations | June 2nd
+
+Footage shows a police van pull up to an intersection, blocking one path. Several officers exit the van and approach a pickup truck stopped at the intersection. Officers order the driver and passengers to exit the car and lay on the ground. Officers then arrest them.
+
+tags: vehicle, arrest
+
+id: ca-losangeles-29
+
+**Links**
+
+* https://twitter.com/davidgallardo/status/1268029756722655232
+* https://twitter.com/greg_doucette/status/1268049956830695427
```

Comment:
this is clearly fucked. but the lack of context and protest presence makes me worry the door will be open for haters to say that this was an arrest of actual criminals

Reply:
Yeah I think that's fair. This was a tricky one I spent poking around for a while to determine cause. But I'm fine with removing it